package.json

```diff
@@ -26,7 +26,7 @@
     },
     "homepage": "https://github.com/chronologic/ethereum-alarm-clock#readme",
     "dependencies": {
-        "@optionality.io/clone-factory": "^0.1.2",
+        "@optionality.io/clone-factory": "0.1.6",
```

Comment:
Is there are reason we are keeping this npm package?

Reply:
For reference, I just copied it from there.

Reply:
```